src/panels/config/integrations/ha-domain-integrations.ts

```diff
@@ -48,6 +48,7 @@ class HaDomainIntegrations extends LitElement {
                 hasMeta
               >
                 <img
+                  alt=""
```

Comment:
There is text accompanying this image, therefore I believe the image to be decorative.